### ISSUE TITLE
change language to indonesian wiki

### ISSUE THREAD

**Issue by harikmarthin** [iamjake648/jasper-wiki#3]
can i change wikipedia url ? when i change with indonesian wiki i get error of the json data. no key for read extract. this is for an example from indonesian wiki  :

`{"batchcomplete":"","query":{"normalized":[{"from":"indonesia","to":"Indonesia"}],"pages":{"44":{"pageid":44,"ns":0,"title":"Indonesia","extract":"Republik Indonesia, disingkat RI atau Indonesia, adalah negara di Asia Tenggara yang dilintasi garis khatulistiwa dan berada di antara benua Asia dan Australia serta antara Samudra Pasifik dan Samudra Hindia. Indonesia adalah negara kepulauan terbesar di dunia yang terdiri dari 13.466 pulau, nama alternatif yang biasa dipakai adalah Nusantara. Dengan populasi lebih dari 258 juta jiwa pada tahun 2016, Indonesia adalah negara berpenduduk terbesar keempat di dunia dan negara yang berpenduduk Muslim terbesar di dunia, dengan lebih dari 207 juta jiwa. Bentuk pemerintahan Indonesia adalah republik, dengan Dewan Perwakilan Rakyat, Dewan Perwakilan Daerah dan Presiden yang dipilih secara langsung.\nIbu kota negara ialah Jakarta. Indonesia berbatasan darat dengan Malaysia di Pulau Kalimantan, dengan Papua Nugini di Pulau Papua dan dengan Timor Leste di Pulau Timor. Negara tetangga lainnya adalah Singapura, Filipina, Australia, dan wilayah persatuan Kepulauan Andaman dan Nikobar di India.\nSejarah Indonesia banyak dipengaruhi oleh bangsa lainnya. Kepulauan Indonesia menjadi wilayah perdagangan penting setidaknya sejak abad ke-7, yaitu ketika Kerajaan Sriwijaya di Palembang menjalin hubungan agama dan perdagangan dengan Tiongkok dan India. Kerajaan-kerajaan Hindu dan Buddha telah tumbuh pada awal abad Masehi, diikuti para pedagang yang membawa agama Islam, serta berbagai kekuatan Eropa yang saling bertempur untuk memonopoli perdagangan rempah-rempah Maluku semasa era penjelajahan samudra. Setelah berada di bawah penjajahan Belanda, Indonesia yang saat itu bernama Hindia Belanda menyatakan kemerdekaannya di akhir Perang Dunia II. Selanjutnya Indonesia mendapat berbagai hambatan, ancaman dan tantangan dari bencana alam, korupsi, separatisme, proses demokratisasi dan periode perubahan ekonomi yang pesat.\nDari Sabang sampai Merauke, Indonesia terdiri dari berbagai suku bangsa, bahasa, dan agama. Berdasarkan rumpun bangsa (ras), Indonesia terdiri atas bangsa asli pribumi yakni Mongoloid Selatan/Austronesia dan Melanesia di mana bangsa Austronesia yang terbesar jumlahnya dan lebih banyak mendiami Indonesia bagian barat. Secara lebih spesifik, suku bangsa Jawa adalah suku bangsa terbesar dengan populasi mencapai 41,7% dari seluruh penduduk Indonesia. Semboyan nasional Indonesia, \"Bhinneka tunggal ika\" (\"Berbeda-beda namun tetap satu\"), berarti keberagaman yang membentuk negara. Selain memiliki populasi padat dan wilayah yang luas, Indonesia memiliki wilayah alam yang mendukung tingkat keanekaragaman hayati terbesar kedua di dunia.\nIndonesia juga anggota dari PBB dan satu-satunya anggota yang pernah keluar dari PBB, yaitu pada tanggal 7 Januari 1965, dan bergabung kembali pada tanggal 28 September 1966 dan Indonesia tetap dinyatakan sebagai anggota yang ke-60, keanggotaan yang sama sejak bergabungnya Indonesia pada tanggal 28 September 1950. Selain PBB, Indonesia juga merupakan anggota dari ASEAN, KAA, APEC, OKI, G-20 dan akan menjadi anggota dari OECD.\n\n"}}}}`

for en url you get some keys, like this :
`{"batchcomplete":"","query":{"normalized":[{"from":"indonesia","to":"Indonesia"}],"pages":{"14579":{"pageid":14579,"ns":0,"title":"Indonesia","extract":"Indonesia **(/\u02cc\u026and\u0259\u02c8ni\u02d0\u0292\u0259/ IN-d\u0259-NEE-zh\u0259 or /\u02cc\u026ando\u028a\u02c8ni\u02d0zi\u0259/ IN-doh-NEE-zee-\u0259; Indonesian: [\u026andonesia]),** officially the Republic of Indonesia (Indonesian: Republik Indonesia [r\u025bpublik \u026andonesia]), is a country in Southeast Asia. Situated between the Indian and Pacific Ocean, it is the largest island country, with more than thirteen thousand islands. It has an estimated population of over 258 million people and is the world's fourth most populous country as well as the most populous Muslim majority country. The world's most populous island of Java contains more than half of the country's population.\nIndonesia's republican form of government includes an elected legislature and president. Indonesia has 34 provinces, of which five have Special Administrative status. Its capital city is Jakarta. The country shares land borders with Papua New Guinea, East Timor, and Malaysia. Other neighbouring countries include Singapore, the Philippines, Australia, Palau, and the Indian territory of the Andaman and Nicobar Islands. Indonesia is a founding member of ASEAN and a member of the G-20 major economies. The Indonesian economy is the world's 16th largest by nominal GDP and the 8th largest by GDP at PPP.\nThe Indonesian archipelago has been an important trade region since at least the 7th century, when Srivijaya and then later Majapahit traded with China and India. Local rulers gradually absorbed foreign cultural, religious and political models from the early centuries CE, and Hindu and Buddhist kingdoms flourished. Indonesian history has been influenced by foreign powers drawn to its natural resources. Muslim traders and Sufi scholars brought the now-dominant Islam, while European powers brought Christianity and fought one another to monopolise trade in the Spice Islands of Maluku during the Age of Discovery. Following three and a half centuries of Dutch colonialism starting from the East Indonesia of West Papua, Timor to eventually all of West Indonesia, at times interrupted by Portuguese, French and British rule, Indonesia secured its independence after World War II. Indonesia's history has since been turbulent, with challenges posed by natural disasters, mass slaughter, corruption, separatism, a democratisation process, and periods of rapid economic change.\nIndonesia consists of hundreds of distinct native ethnic and linguistic groups. The largest \u2013 and politically dominant \u2013 ethnic group are the Javanese. A shared identity has developed, defined by a national language, ethnic diversity, religious pluralism within a Muslim-majority population, and a history of colonialism and rebellion against it. Indonesia's national motto, \"Bhinneka Tunggal Ika\" (\"Unity in Diversity\" literally, \"many, yet one\"), articulates the diversity that shapes the country. Despite its large population and densely populated regions, Indonesia has vast areas of wilderness that support the world's second highest level of biodiversity. The country has abundant natural resources like oil and natural gas, tin, copper and gold. Agriculture mainly produces rice, tea, coffee, spices and rubber. Indonesia's major trading partners are Japan, the United States and the surrounding nations of Singapore, Malaysia and Australia."}}}}`

cay you help me ? i'm beginner. thanks

**Comment by iamjake648** [comment]
Yeah, you should be able to change - may have to change the path the program uses to parse the values from JSON.
